### PR TITLE
Update OpenStackDataPlane vars

### PR DESCRIPTION
**Pull request by fao89**
dataplane is successfully deployed:
```
[dpadev@osp-dev-05 director_standalone]$ oc get osdp
NAME        STATUS   MESSAGE
openstack   True     DataPlane ready
```
but standalone host isn't showing up:
```
[dpadev@osp-dev-05 director_standalone]oc rsh openstackclient
sh-5.1$ openstack compute service list

+--------------------------------------+----------------+------------------------+----------+---------+-------+----------------------------+
| ID                                   | Binary         | Host                   | Zone     | Status  | State | Updated At                 |
+--------------------------------------+----------------+------------------------+----------+---------+-------+----------------------------+
| dd910f7f-de46-477d-9642-2e7e6864a05f | nova-conductor | nova-cell1-conductor-0 | internal | enabled | up    | 2023-05-05T19:35:51.000000 |
| 429c1d45-63fb-456d-86d6-3a992699e899 | nova-conductor | nova-cell0-conductor-0 | internal | enabled | up    | 2023-05-05T19:35:55.000000 |
| 34e7b894-9533-4216-b5fb-b8bd5b834804 | nova-scheduler | nova-scheduler-0       | internal | enabled | up    | 2023-05-05T19:35:57.000000 |
+--------------------------------------+----------------+------------------------+----------+---------+-------+----------------------------+

sh-5.1$ openstack network agent list

+--------------------------------------+----------------------+--------------------+-------------------+-------+-------+----------------+
| ID                                   | Agent Type           | Host               | Availability Zone | Alive | State | Binary         |
+--------------------------------------+----------------------+--------------------+-------------------+-------+-------+----------------+
| 77eedc86-cdcf-48bd-9abf-29902ecfe1c5 | OVN Controller agent | crc-9ltqk-master-0 |                   | :-)   | UP    | ovn-controller |
+--------------------------------------+----------------------+--------------------+-------------------+-------+-------+----------------+

sh-5.1$ openstack hypervisor list


```
trying to fix this by updating the ansible vars

### CODE REVIEW

**Comment by fao89** [comment]
I suspect I'm missing some install_yamls network magic, @jistr @fultonj do you have any clue on what might be causing this?

**Review by jistr — APPROVED** [review]
Looks good, this part isn't yet enabled in the test suite so maybe we could merge even though the CI is red. Let's discuss on the call?

**Review by fmount — APPROVED** [review]

**Comment by fultonj** [comment]
> dataplane is successfully deployed:
> 
> ```
> [dpadev@osp-dev-05 director_standalone]$ oc get osdp
> NAME        STATUS   MESSAGE
> openstack   True     DataPlane ready
> ```
> 
> but standalone host isn't showing up:
> 
> ```
> [dpadev@osp-dev-05 director_standalone]oc rsh openstackclient
> sh-5.1$ openstack compute service list
> 
> +--------------------------------------+----------------+------------------------+----------+---------+-------+----------------------------+
> | ID                                   | Binary         | Host                   | Zone     | Status  | State | Updated At                 |
> +--------------------------------------+----------------+------------------------+----------+---------+-------+----------------------------+
> | dd910f7f-de46-477d-9642-2e7e6864a05f | nova-conductor | nova-cell1-conductor-0 | internal | enabled | up    | 2023-05-05T19:35:51.000000 |
> | 429c1d45-63fb-456d-86d6-3a992699e899 | nova-conductor | nova-cell0-conductor-0 | internal | enabled | up    | 2023-05-05T19:35:55.000000 |
> | 34e7b894-9533-4216-b5fb-b8bd5b834804 | nova-scheduler | nova-scheduler-0       | internal | enabled | up    | 2023-05-05T19:35:57.000000 |
> +--------------------------------------+----------------+------------------------+----------+---------+-------+----------------------------+
> 
> sh-5.1$ openstack network agent list
> 
> +--------------------------------------+----------------------+--------------------+-------------------+-------+-------+----------------+
> | ID                                   | Agent Type           | Host               | Availability Zone | Alive | State | Binary         |
> +--------------------------------------+----------------------+--------------------+-------------------+-------+-------+----------------+
> | 77eedc86-cdcf-48bd-9abf-29902ecfe1c5 | OVN Controller agent | crc-9ltqk-master-0 |                   | :-)   | UP    | ovn-controller |
> +--------------------------------------+----------------------+--------------------+-------------------+-------+-------+----------------+
> 
> sh-5.1$ openstack hypervisor list
> ```
> 
> trying to fix this by updating the ansible vars

As per our chat, we believe this is happening because the edpm-node cannot reach the network where rabbit is running.

File: tests/roles/dataplane_adoption/tasks/main.yaml
Comment (jistr) [reply tests/roles/dataplane_adoption/tasks/main.yaml:220]: None of the `dataplane_adoption` role is currently included in the CI playbook (the role is commented out for now), so the job won't tell us. But in theory i think it should work:

```
$ cat <<EOF
> asdf $(echo ghij)
> EOF
asdf ghij
```

The `oc` command will be authenticated the same way as the `oc apply -f <<EOF` that you quoted -- the `{{ oc_header }}` can add `oc` into `$PATH` if it wasn't there already (depending on execution environment), and at the beginning of the test suite we explicitly authenticate, ensuring cached authentication for the rest of the test suite run.

Comment (fultonj) [reply tests/roles/dataplane_adoption/tasks/main.yaml:220]: I thought we'd see the output of the CI job in order to confirm before merging. But since it's not in CI yet it can't break CI, so we're safe to merge.

**Review by fultonj — APPROVED** [review]
/lgtm 

+2

**Comment by jistr** [comment]
I forgot to press the merge button here, doing so now :)

**Pull request merged**
